test/backend/QueryScheduler.spec.js

```diff
@@ -66,7 +67,7 @@ describe('QueryScheduler', function() {
 
         assert(spy.notCalled, 'job should not have been called yet');
 
-        return wait(1.5 * refreshInterval * 1000)
+        return wait(1.2 * refreshInterval * 1000)
```

Comment:
why?

Reply:
I haven't looked into how the `node-schedule` scheduling algorithm is implemented, but at the resolution of a few seconds, I was having trouble consistently producing the same run count with the `1.5` multiplier. Note, this is not an issue with any intervals on the order of what we support in an actual, non-test environment.

I have since updated the tests to use fake timers and test on a more representative case which resolves this problem (the `1.5` multiplier has been restored).